### ISSUE TITLE
Missing Cypress, STON, and Tonel packages (V1.2.11)

### ISSUE THREAD

**Issue by ericwinger** [GemTalk/Rowan#728]
Using this script to install Rowan V1.2.11 into 3.6.1 loads classes into Rowan project but Cypress, STON, and Tonel packages are missing. Presumably, the classes are installed because Jadeite 3.0.91.1-pre-release1 is working. 
```
input $ROWAN_PROJECTS_HOME/Rowan/platforms/gemstone/topaz/bootstrap/bootstrapRowan.gs
```

![image](https://user-images.githubusercontent.com/36212516/125839320-6dbd6321-4fa2-4175-864e-162fb431c2ac.png)
![image](https://user-images.githubusercontent.com/36212516/125839346-3e0e3607-8839-48a4-a8f4-2f78047a7560.png)
![image](https://user-images.githubusercontent.com/36212516/125839381-1523e0c8-c38d-42a5-bbf0-39415f674625.png)
![image](https://user-images.githubusercontent.com/36212516/125839413-40561504-a033-43e2-9e6f-7cb633eb4b3e.png)

**Comment by ericwinger** [comment]
Based on the commit changes, looks like the problem was a minor capitalization problem in the paths.

**Comment by dalehenrich** [comment]
The bootstrap .gs file for Rowan includes test classes (like TonelWriterForCypressTest), however, the components/attributes that were used to adopt the code, did not include test classes have remained unpackaged leading to the following error reported by @ericwinger hen attempting to reload the Tonel project:
```
a UserDefinedError occurred (error 2318), reason:halt, No loaded project named '(NONE)' found
[1] UserDefinedError (AbstractException) >> _signalWith: @5 line 25
[2] UserDefinedError (AbstractException) >> signal @2 line 47
[3] RwGsImage class (Object) >> error: @6 line 7
[4] [] in RwGsImage class >> loadedProjectNamed: @9 line 5
[5] [] in RwGsImage class >> loadedProjectNamed:ifAbsent: @19 line 13
[6] StringKeyValueDictionary >> at:ifAbsent: @14 line 10
[7] RwGsImage class >> loadedProjectNamed:ifAbsent: @5 line 8
[8] RwGsImage class >> loadedProjectNamed: @2 line 4
[9] [] in RwPrjLoadTool >> _doProjectSetLoad_254:instanceMigrator:originalProjectSet:processedClassNames: @25 line 12
[10] RwExistingVisitorAddingExistingClassNotification (AbstractException) >> _executeHandler: @7 line 11
[11] RwExistingVisitorAddingExistingClassNotification (AbstractException) >> _signalWith: @1 line 2
[12] RwExistingVisitorAddingExistingClassNotification (AbstractException) >> signal @2 line 47
[13] RwGsPatchSet_254 >> addClassModification:toPatchSetInPackage:inProject: @11 line 10
[14] RwClassModification >> addModificationToPatchSet:inPackage:inProject: @2 line 4
[15] RwGsImagePatchVisitor_254 >> visitClassModification: @2 line 4
[16] RwClassModification >> acceptVisitor: @2 line 3
[17] [] in RwGsImagePatchVisitor_254 >> visitClassesModification: @7 line 4
[18] Dictionary >> do: @8 line 12
[19] RwGsImagePatchVisitor_254 >> visitClassesModification: @3 line 4
[20] RwClassesModification >> acceptVisitor: @2 line 3
[21] RwGsImagePatchVisitor_254 >> visitPackageModification: @20 line 14
[22] RwPackageModification >> acceptVisitor: @2 line 3
[23] [] in RwGsImagePatchVisitor_254 >> visitPackagesModification: @7 line 3
[24] Dictionary >> do: @8 line 12
[25] RwGsImagePatchVisitor_254 >> visitPackagesModification: @3 line 3
[26] RwPackagesModification >> acceptVisitor: @2 line 3
[27] RwGsImagePatchVisitor_254 >> visitProjectModification: @31 line 23
[28] RwProjectModification >> acceptVisitor: @2 line 3
[29] [] in RwGsImagePatchVisitor_254 >> visitProjecteSetModification: @19 line 8
[30] Dictionary >> do: @8 line 12
[31] RwGsImagePatchVisitor_254 >> visitProjecteSetModification: @7 line 8
[32] RwProjectSetModification >> acceptVisitor: @2 line 3
[33] RwGsImagePatchVisitor_254 >> visit: @2 line 3
[34] RwGsImage class >> applyModification_254:visitorClass:instanceMigrator: @4 line 5
[35] RwGsImage class >> applyModification_254:instanceMigrator: @8 line 12
[36] RwPrjLoadTool >> _loadProjectSetDefinition_254:instanceMigrator: @17 line 18
[37] [] in RwPrjLoadTool >> _doProjectSetLoad_254:instanceMigrator:originalProjectSet:processedClassNames: @7 line 4
[38] ExecBlock0 (ExecBlock) >> on:do: @3 line 44
[39] RwPrjLoadTool >> _doProjectSetLoad_254:instanceMigrator:originalProjectSet:processedClassNames: @2 line 6
[40] RwPrjLoadTool >> loadProjectNamed:instanceMigrator: @7 line 9
[41] RwPrjLoadTool >> loadProjectNamed: @4 line 5
[42] [] in RowanProjectService >> reloadProject @12 line 2
[43] ExecBlock0 (ExecBlock) >> on:do: @3 line 44
[44] RowanProjectService >> reloadProject @2 line 3
[45] RowanProjectService (Object) >> perform:withArguments: @1 line 12
[46] RowanProjectService (RowanService) >> servicePerform:withArguments: @7 line 14
[47] RowanProjectService >> servicePerform:withArguments: @6 line 4
[48] [] in JadeServer >> updateFromSton: @30 line 12
[49] OrderedCollection (Collection) >> do: @5 line 10
[50] [] in JadeServer >> updateFromSton: @19 line 8
[51] ExecBlock0 (ExecBlock) >> on:do: @3 line 44
[52] JadeServer64bit35 (JadeServer) >> updateFromSton: @8 line 13
[53] GsNMethod class >> _gsReturnToC @1 line 11
===============================================
```
- [ ] decide whether or not tests should be included in the bootstrap .gs file for Rowan (probably not)
- [ ] add an audit step at the end of the bootstrap script to reveal issues like this one

**Comment by dalehenrich** [comment]
The `adoptRowan.gs` script already does an audit at the end of the load, so the final tests is to try to manually do a load ...

**Comment by dalehenrich** [comment]
looks like `STONWriterTests debug: #testDateAndTime` is failing for 3.6.1 .. not a surprise, because of SmallDateAndTime, so that STON extensions for 3.6.1 are needed as well ...

**Comment by dalehenrich** [comment]
turns out that the STON extensions are included in 3.6.2 kernel (they are there to support the tmethod and tfile topaz commands), so all that remains is to change the test ... default printString format for DateAndTime changed in 3.6.2 and for STON it is not worth fiddling with the code to preserve the printString format that was there previously ...

I think that takes care of the current known issues
